packages/api-client/collections.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
```

Comment:
This, and all the api-client/* methods were copied wholesale from integration-test/api/* where possible with only minor interface changes.     There is a follow-on ticket to pull the remaining methods (where appropriate) out of integration-test/api/api.js, and consider a possible refactor at that time.

Reply:
Relevant ticket: https://bugs.earthdata.nasa.gov/browse/CUMULUS-1809